disnake/http.py

```diff
@@ -244,13 +255,16 @@ def __init__(
         self.proxy_auth: Optional[aiohttp.BasicAuth] = proxy_auth
         self.use_clock: bool = not unsync_clock
 
-        user_agent = "DiscordBot (https://github.com/DisnakeDev/disnake {0}) Python/{1[0]}.{1[1]} aiohttp/{2}"
-        self.user_agent: str = user_agent.format(__version__, sys.version_info, aiohttp.__version__)
+        self.user_agent: str = get_user_agent()
```

Comment:
If someone were to modify this value after the session is created, the change wouldn't apply. Definitely an edge case (in addition to being undocumented), but may still be something worth consideration.

Reply:
Fair, we might be able to remove the user_agent attribute altogether or turn it into a property (as it wouldn't add much to runtime loss

Reply:
Won't change this, just something to note that the user_agent being changed doesn't affect the session unless it is recreated.

Reply:
https://github.com/DisnakeDev/disnake/pull/959/commits/e671bc2b8c0ed3e42c4cdb5f89caf8a028218596